src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,476 @@
+    examples:
+        - name: Creates Authorization rules for Queue
+          text: az servicebus queue authorizationrule create --resource-group myresourcegroup --namespace-name mynamespace
+           --queue-name myqueue --name myauthorule --access-rights [Listen]
+"""
+
+helps['servicebus queue authorizationrule show'] = """
+    type: command
+    short-summary: shows the details of AuthorizationRule
+    examples:
+        - name: shows the details of AuthorizationRule
+          text: az servicebus queue authorizationrule show --resource-group myresourcegroup --namespace-name mynamespace
+           --queue-name myqueue --name myauthorule
+"""
+
+helps['servicebus queue authorizationrule list'] = """
+    type: command
+    short-summary: shows the list of AuthorizationRule by Queue
+    examples:
+        - name: shows the list of AuthorizationRule by Queue
+          text: az servicebus queue authorizationrule show --resource-group myresourcegroup --namespace-name mynamespace
+           --queue-name myqueue
+"""
+
+helps['servicebus queue authorizationrule keys list'] = """
+    type: command
+    short-summary: Shows the connectionstrings of AuthorizationRule for the Queue.
+    examples:
+        - name: Shows the connectionstrings of AuthorizationRule for the queue.
+          text: az servicebus queue authorizationrule list-keys --resource-group myresourcegroup --namespace-name mynamespace
+           --queue-name myqueue --name myauthorule
+"""
+
+helps['servicebus queue authorizationrule keys renew'] = """
+    type: command
+    short-summary: Regenerate the connection strings of AuthorizationRule for the namespace.
+    examples:
+        - name: Regenerate the connectionstrings of AuthorizationRule for the namespace.
+          text: az servicebus queue authorizationrule regenerate-keys --resource-group myresourcegroup --namespace-name mynamespace
+           --queue-name myqueue --name myauthorule --key PrimaryKey
+"""
+
+helps['servicebus queue authorizationrule delete'] = """
+    type: command
+    short-summary: Deletes the AuthorizationRule of the Queue.
+    examples:
+        - name: Deletes the AuthorizationRule of the queue.
+          text: az servicebus queue authorizationrule delete --resource-group myresourcegroup --namespace-name mynamespace
+           --queue-name myqueue --name myauthorule
+"""
+
+helps['servicebus topic create'] = """
+    type: command
+    short-summary: Creates the Service Bus Topic
+    examples:
+        - name: Create a new queue.
+          text: az servicebus topic create --resource-group myresourcegroup --namespace-name mynamespace --name {topicname}
+"""
+
+helps['sb topic show'] = """
+    type: command
+    short-summary: Shows the Service Bus Topic Details
+    examples:
+        - name: Shows the Topic details.
+          text: az sb topic get --resource-group myresourcegroup --namespace-name mynamespace --name {topicname}
```

Comment:
Same here.
And there is no `az sb topic show` command.

Reply:
Resolved